cea/tests/test_inputs_setup_workflow.py

```
@@ -4,7 +4,7 @@
 
 import cea.config
 from cea.utilities import create_polygon
-from cea.datamanagement import zone_helper, surroundings_helper, terrain_helper, streets_helper, data_initializer, \
+from cea.datamanagement import zone_helper, surroundings_helper, terrain_helper, streets_helper, database_helper, \
```

Ruff (0.8.2)

`cea.datamanagement.database_helper` imported but unused

Remove unused import: `cea.datamanagement.database_helper`

(F401)